spanner/tests/system/utils/populate_streaming.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Populate spanner databases with data for streaming system tests."""
+
+import os
+
+from google.cloud.spanner import Client
+from google.cloud.spanner.keyset import KeySet
+from google.cloud.spanner.pool import BurstyPool
+
+
+INSTANCE_NAME = 'gcp-streaming-systests'
+DATABASE_NAME = 'testing'
```

Comment:
When running a script from another directory, `sys.path[0]` is *that* directory, rather than the CWD (which is not on the path anywhere in that case, unless one sets `PYTHONPATH`).  We therefore cannot import `tests.system.utils.foo` within such scripts, unless we set `sys.path` ourselves.